pytorch/Dockerfile

```diff
@@ -181,11 +182,15 @@ RUN apt-get update && \
     intel-oneapi-runtime-mkl=${MKL_VER} \
     intel-oneapi-runtime-ccl=${CCL_VER};
 
+RUN rm -rf /etc/apt/sources.list.d/intel-gpu-jammy.list /etc/apt/sources.list.d/oneAPI.list
+
+FROM ipex-xpu-base AS ipex-xpu-base-req
```

Comment:
Can we change the stage names to be more description, for instance having `xpu-base` and then `ipex-xpu-base` better describes when ipex gets installed.

Reply:
modified to better reflect the stage's purpose